### DISCUSSION TITLE
Separating multiple delimiters in dplyr

### DISCUSSION THREAD

**Discussion by sr320** [RobertsLab/resources#1298] [Q&A] [answered]
I feel like I have asked before, but could not find.. 

In R I want to separate SPIDs in Blast output
![Screen Shot on 2021-09-22 at 15-03-48](https://user-images.githubusercontent.com/4442039/134428518-88d97323-960e-445b-afea-edd9578d0b68.png)

How do I get V2 into three columns using dplyr / tidyverse?

**Comment by aspencoyle** [comment, marked as answer]
Ooh, I got this - ran into this exact problem recently! Something along these lines should work!

`blast_sep <-  separate(data = blast, col = V2,
                          into = c("sp", "Accession_ID", "Gene"), 
                          sep = "\\|")`

**Reply by sr320** [reply]
Awesome! -- 
and with pipes 

```
blast <- read.csv("../data/cbai_transcriptome_v3.1.blastx.outfmt6.txt", header = FALSE, sep="\t") %>% 
    separate(col = V2, into = c("sp", "Accession_ID", "Gene"), sep = "\\|")
```
